app/services/physical_holdings_markup_builder.rb

```diff
@@ -237,8 +237,15 @@ def self.open_location?(location)
   end
 
   def self.requestable_location?(location, adapter, holding)
-    return false if location.nil? || adapter.unavailable_holding?(holding)
-    location[:requestable]
+    if location.nil?
+      false
+    elsif adapter.unavailable_holding?(holding)
+      false
+    elsif Orangelight.read_only_mode
+      false
+    else
+      location[:requestable]
+    end
   end
 
```

Comment:
can we use the guard clause with multiple ors instead of multiple lines with false?
`return false if location.nil? || adapter.unavailable_holding?(holding) || Orangelight.read_only_mode`
`location[:requestable]`

Reply:
To me it's more readable to have each condition on its own line.

@eliotjordan is likely trying this in response to some discussions we've had in refactorama about postfix conditionals. It turns out that they hide test coverage holes because even if the condition is not tested the line is hit. Because of that, I would vote for the form here.

Reply:
That was exactly my thinking here. Granted it's not perfect, but the entire Class needs a refactor and hopefully this logic can be simplified at that time.